src/components/AudioCell.vue

```diff
@@ -115,9 +115,13 @@ const props =
 
 const emit =
   defineEmits<{
+    // focusTarget：
+    //   textField: テキストフィールドにフォーカス。特別な理由がない限りはこちらを使う。
+    //   root: AudioCell自体にフォーカス。テキストフィールドにはフォーカスしない。
+    //         selectedAudioKeysをactiveAudioKeyだけにしたくない時に使う。
```

Comment:
文脈を知らない人にもそこそこ伝わり、かつ文量もそこまで多くない形だと･･･こんな感じ･･･？
```suggestion
    // focusTarget：
    //   textField: テキストフィールドにフォーカス。複数選択が解除される。特別な理由がない限りはこちらを使う。
    //   root: AudioCell自体にフォーカス。こちらは複数選択を解除しない。
```